tensorflow/compiler/xla/service/gpu/cudnn_conv_rewriter.cc

```diff
@@ -155,6 +153,15 @@ std::tuple<bool, Window, ConvolutionDimensionNumbers> MatchBackwardFilter(
                "to fold it to a backward filter convolution.";
     return no_match_result;
   }
+  auto rhs_in =
```

Comment:
Although logically removing the condition `!window_util::HasWindowDilation(conv->window())` should just work, I haven't been able to completely access the scope of making that change. Hence I am a more non-invasive check that isolates depthwise fwd convs and addresses the failures. This also uses the underlying condition that for a backward filter conv, the input and the output dims cannot be the same.